.travis.yml

```diff
@@ -8,9 +8,13 @@ cache:
 
 # don't cache our own libraries generated in the build
 # so they always get rebuilt properly and to avoid init issues
+# | true with unregister as they might not exist on build errors
 before_cache:
   - cd $HOME/elektra-cabal-sandbox
-  - cabal sandbox hc-pkg unregister spectranslator specelektra libfa libelektra-haskell
+  - cabal sandbox hc-pkg unregister spectranslator | true
```

Comment:
Typo: `|` → `||`

While piping the output of `cabal sandbox` to `true`, will also work here – by default the shell  ignores the exit status of a command to the left of a pipe – I think you want to run `true` if (and only if) `cabal sandbox` fails. 

I could be wrong though and you always want to ignore the output of `cabal sandbox`. In that case I think the command is a little bit to „clever” 😊.

Reply:
yes i don't really care if it got created during the build unregister it if not it will fail and well then not. so returning true in any case is fine

Reply:
In that case, please write something like

```bash
cabal sandbox hc-pkg unregister spectranslator > /dev/null; true
```

instead. Otherwise 

- stupid people like me will assume the command is incorrect,
- the command will fail, if some sets the option `pipefail` and `cabal sandbox` fails

.